### PR TITLE
Properly handle `--project` in CLI error messages

### PR DESCRIPTION
**Pull request by bensmrs**
This PR fixes a problem where, in case of an error, the CLI would prompt the user to run an invalid command if they had specified a project other than the default one.

Before:
```
#> incus start debian --project test
Error: Failed to run: <...>
Try `incus info --show-log debian` for more info
```

After:
```
#> incus start debian --project test
Error: Failed to run: <...>
Try `incus info --show-log debian --project test` for more info
```

### CODE REVIEW

**Comment by bensmrs** [comment]
Quick warning: the i18n strings should now contain an additional `%s`, which needs to be taken into account by translators

File: cmd/incus/action.go [hunk 1]
```diff
@@ -331,7 +331,12 @@ func (c *cmdAction) doAction(action string, conf *config.Config, nameArg string)
 	err = cli.CancelableWait(op, &progress)
 	if err != nil {
 		progress.Done("")
-		return fmt.Errorf("%s\n"+i18n.G("Try `incus info --show-log %s` for more info"), err, nameArg)
+		projectArg := ""
+		if conf.ProjectOverride != "" && conf.ProjectOverride != "default" {
+			projectArg = " --project " + conf.ProjectOverride
+		}
```
Comment (stgraber) [inline cmd/incus/action.go:337]: I'd use `api.ProjectDefaultName` instead of the hardcoded `"default"` in here.

File: cmd/incus/launch.go [hunk 2]
```diff
@@ -143,7 +143,12 @@ func (c *cmdLaunch) Run(cmd *cobra.Command, args []string) error {
 			prettyName = fmt.Sprintf("%s:%s", remote, name)
 		}
 
-		return fmt.Errorf("%s\n"+i18n.G("Try `incus info --show-log %s` for more info"), err, prettyName)
+		projectArg := ""
+		if conf.ProjectOverride != "" && conf.ProjectOverride != "default" {
```
Comment (stgraber) [inline cmd/incus/launch.go:147]: Same as above

File: cmd/incus/action.go (hunk 1 above)
Comment (bensmrs) [reply cmd/incus/action.go:337]: Noted. FYI, `"default"` is hardcoded in many places in `cmd/`.

Comment (stgraber) [reply cmd/incus/action.go:337]: Yeah, it's one of those where we're trying to not introduce new ones at least :)

**Comment by bensmrs** [comment]
Should be fine now

**Review by stgraber — APPROVED** [review]

**Pull request merged**